Toggl.Foundation.MvvmCross/ViewModels/SelectDateTimeViewModel.cs

```diff
@@ -12,53 +13,43 @@ namespace Toggl.Foundation.MvvmCross.ViewModels
     [Preserve(AllMembers = true)]
     public class SelectDateTimeViewModel : MvxViewModel<DateTimePickerParameters, DateTimeOffset>
     {
-        private DateTimeOffset defaultResult;
-
         private readonly IMvxNavigationService navigationService;
 
-        private DateTimeOffset currentDateTime;
-        public DateTimeOffset CurrentDateTime
-        {
-            get => currentDateTime;
-            set
-            {
-                var newValue = createDateTimeBasedOnMode(value);
-                if (currentDateTime == newValue) return;
+        private DateTimeOffset defaultResult;
 
-                currentDateTime = newValue.Clamp(MinDate, MaxDate);
+        public BehaviorSubject<DateTimeOffset> SelectedDateTime = new BehaviorSubject<DateTimeOffset>(DateTimeOffset.Now);
```

Comment:
No public _fields_. Turn this into a _property_:
```suggestion
        public BehaviorSubject<DateTimeOffset> SelectedDateTime { get; } = new BehaviorSubject<DateTimeOffset>(DateTimeOffset.Now);
```